build-system/tasks/css/index.js

```diff
@@ -84,80 +78,77 @@ const cssEntryPoints = [
   },
 ];
 
+/**
+ * Copies the css from the build folder to the dist folder
+ */
+async function copyCss() {
+  const startTime = Date.now();
+  for (const {outCss} of cssEntryPoints) {
+    await fs.copy(`build/css/${outCss}`, `dist/${outCss}`);
+  }
```

Comment:
Nit: parallelize for speeeeeeed
```suggestion
  await Promise.all(cssEntryPoints.map(({outCss}) => {
    return fs.copy(`build/css/${outCss}`, `dist/${outCss}`);
  }));
```

Reply:
lol, i had the same comment 🚀

Reply:
Whaddayaknow, I cannot possibly say no to good ideas from both of you, so here I am sitting, addressing a bunch of other comments from Jake's first review that I initially thought I'd punt on 😃

Reply:
Parallelizing the top-level CSS copying results in this race caught by `gulp check-sourcemaps` with extra logging: https://app.circleci.com/pipelines/github/ampproject/amphtml/2953/workflows/93cf8701-336b-4a00-b4cd-332a8076f574/jobs/34479

I've reverted it for now, can optimize in a future refactor.